dex-it-common/src/main/scala/com/wavesplatform/dex/it/docker/PortBindingKeeper.scala

```diff
@@ -0,0 +1,81 @@
+package com.wavesplatform.dex.it.docker
+
+import cats.implicits._
+import com.github.dockerjava.api.command.CreateContainerCmd
+import com.github.dockerjava.api.model.Ports.Binding
+import com.github.dockerjava.api.model.{ExposedPort, PortBinding}
+import com.wavesplatform.dex.domain.utils.ScorexLogging
+
+import java.net.ServerSocket
+import java.util
+import scala.jdk.CollectionConverters._
+import scala.util.{Random, Using}
+
+object PortBindingKeeper extends ScorexLogging {
```

Comment:
Also, it looks like we don't use `Either`, we just throw exceptions which are inside `Left(ex)`. Probably, it'd better to just throw exceptions.